solr/solrj/src/test/org/apache/solr/client/solrj/SolrExampleTests.java

```diff
@@ -1831,7 +1831,7 @@ public void testChildDoctransformer() throws IOException, SolrServerException {
       
       q = new SolrQuery("q", "*:*", "indent", "true");
       q.setFilterQueries(parentFilter);
-      q.setFields("id,[child parentFilter=\"" + parentFilter +
+      q.setFields("id, level_i, [child parentFilter=\"" + parentFilter +
```

Comment:
These tests fail since the recent changes made in this branch result the child doc transformer defaulting to the top fl. Beforehand, the default child fl was "*", which does not filter "level_i" from child docs. Should we default to "*" in lucene version < 8, and change this behaviour only once Solr 8 arrives?